src/sailor/blank-app/conf/conf.lua

```diff
@@ -11,19 +11,18 @@ local conf = {
 		default_error404 = 'error/404',
 		enable_autogen = false, -- default is false, should be true only in development environment
 		friendly_urls = false,
-		max_upload = 1024 * 1024, 
 		environment = "development",  -- this will use db configuration named development
 		hide_stack_trace = false, -- false recommended for development, true recommended for production
-		enable_admin=true, -- set true only in development. Set password below too.
-		admin_password="test"
+		enable_admin=false, -- set true only in development. Set password below too.
+		admin_password=""
 	},
 
 	db = {
 		development = { -- current environment
-			driver = 'mysql',
-			host = '',
-			user = '',
-			pass = '',
```

Comment:
Yeah. I expected that and I have been meaning to include it in the commit message as a breaking change. The name attribute of the fields are generated with a loop(in dashboard.lp). The Mail settings and the db settings had common fields such as user & pass. This created a nested table in page.POST when I tried looping through it in the controller. I wanted to generate the config file with a loop but I had to add strings in between(string1,string2 etc). So I made the file writing part manual with each value written separately. One thing that can be done is that, the db fields can be reverted back and I will change the smtp fields.